kubernetes_cluster_autoscaler/README.md

```diff
@@ -19,6 +19,59 @@ No additional installation is needed on your server.
 
 2. [Restart the Agent][5].
 
+#### Metric collection
+
+Make sure that the Prometheus-formatted metrics are exposed in your kubernetes_cluster_autoscaler cluster. 
+For the Agent to start collecting metrics, the kubernetes_cluster_autoscaler pods need to be annotated.
+
+Kubernetes Cluster Autoscaler has metrics and livenessProbe endpoints that can be accessed on port 8085 (you can change that using the --address flag). These endpoints are located under `/metrics` and `/health-check` and provide valuable information about the state of your cluster during scaling operations.
```

Comment:
non blocking nit. Is it possible to include the links to the cluster autoscaler docs where these are documented?